domain/src/main/java/org/oppia/android/domain/exploration/lightweightcheckpointing/ExplorationCheckpointController.kt

```diff
@@ -69,65 +68,55 @@ class ExplorationCheckpointController @Inject constructor(
   /**
    * Records an exploration checkpoint for the specified profile.
    *
-   * @return a [DataProvider] that indicates the success/failure of the save operation.
-   *
-   * Success result is returned if the checkpoint is saved successfully. The success result
-   * returned has the value
-   * [ExplorationCheckpointDatabaseState.CHECKPOINT_DATABASE_SIZE_LIMIT_EXCEEDED]
-   * if the database has exceeded the size limit of [explorationCheckpointDatabaseSizeLimit],
-   * otherwise the success result is returned with the value
-   * [ExplorationCheckpointDatabaseState.CHECKPOINT_DATABASE_SIZE_LIMIT_NOT_EXCEEDED].
+   * @return a [Deferred] that upon completion indicates the current state of the checkpoint
+   *         database. If the size of the checkpoint database is less than the allocated limit of
```

Comment:
Continuation is wrong here for at-clause: should be +4 from the '@'.

Reply:
done